src/device_info.sh

```diff
@@ -120,28 +121,39 @@ function get_cpu()
   flag=${flag:-'SILENT'}
   cmd_model="lscpu | grep 'Model name:' | sed --regexp-extended 's/Model name:\s+//g' | cut --delimiter=' ' -f1"
   cmd_frequency="lscpu | grep MHz | sed --regexp-extended 's/(CPU.*)/\t\t\1/'"
+  cmd_architecture="lscpu | grep 'Architecture:' | sed --regexp-extended 's/Architecture:\s+//g' | cut --delimiter=' ' -f1"
```

Comment:
From my previous comment, you can just use `uname --machine`

Reply:
You need to make this change in this commit (second).